arch/arm64/boot/dts/qcom/sdm636-motorola-beckham.dts

```diff
@@ -0,0 +1,744 @@
+		regulator-max-microvolt = <3700000>;
+
+		regulator-always-on;
+		regulator-boot-on;
+	};
+};
+
+&adreno_gpu {
+	status = "okay";
+};
+
+&adreno_gpu_zap {
+	firmware-name = "a512_zap.mdt";
+};
+
+&adsp_pil {
+	firmware-name = "adsp.mdt";
+};
+
+&blsp2_uart1 {
+	status = "okay";
+
+	/* HCI Bluetooth */
+	bluetooth {
+		compatible = "qcom,wcn3990-bt";
+		vddxo-supply = <&vreg_l9a_1p8>;
+		vddrf-supply = <&vreg_l6a_1p3>;
+		vddch0-supply = <&vreg_l19a_3p3>;
+		vddio-supply = <&vreg_l13a_1p8>;
+		max-speed = <3200000>;
+	};
+};
+
+&blsp_i2c2 {
+	status = "okay";
+
+	gpio_expander_1: gpio@20 {
+		compatible = "nxp,pcal6416";
+		reg = <0x20>;
+
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		vcc-supply = <&vreg_l13a_1p8>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_expander_default>;
+
+		reset-gpios = <&tlmm 34 GPIO_ACTIVE_LOW>;
+	};
+
+	gpio_expander_2: gpio@21 {
+		compatible = "nxp,pcal6416";
+		reg = <0x21>;
+
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		vcc-supply = <&vreg_l13a_1p8>;
+/*
+		Both GPIO expanders share a reset pin, Linux can't handle this as far as I know
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_expander_default>;
+
+		reset-gpios = <&tlmm 34 GPIO_ACTIVE_LOW>;
+*/
+	};
+	/* ti,drv2624 @ 0x5a, vibrator */
+	/* analogix,anx7805 @ 0x72 hdmi converter for mods */
+};
+
+&blsp_i2c4 {
+	status = "okay";
+
+	touchscreen@20 {
+		compatible = "syna,rmi4-i2c";
+		reg = <0x20>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		interrupt-parent = <&tlmm>;
+		interrupts = <67 IRQ_TYPE_EDGE_FALLING>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&ts_reset>;
+
+		vdd-supply = <&vreg_l6b_3p3>;
+		vio-supply = <&vreg_l11a_1p8>;
+
+		rmi4-f01@1 {
+			reg = <0x01>;
+			syna,nosleep-mode = <1>;
+		};
+
+		rmi4-f12@12 {
+			reg = <0x12>;
+			syna,sensor-type = <1>;
+		};
+	};
+};
+
+&blsp_i2c6 {
+	status = "okay";
+
+	nfc@28 {
+		compatible = "nxp,nxp-nci-i2c";
+		reg = <0x28>;
+
+		interrupt-parent = <&tlmm>;
+		interrupts = <40 IRQ_TYPE_EDGE_RISING>;
+
+		enable-gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>;
+		firmware-gpios = <&tlmm 61 GPIO_ACTIVE_HIGH>;
+
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&nfc_default>;
+	};
+};
+
+&mdss {
+	status = "okay";
+};
+
+&mdss_dsi0 {
+	status = "okay";
+
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	vdd-supply = <&vreg_l1b_0p925>;
+	vdda-supply = <&vreg_l1a_1p225>;
+
+	panel: panel@0 {
+		reg = <0>;
+		compatible = "samsung,ams601nt12";
+
+		reset-gpios = <&tlmm 53 GPIO_ACTIVE_LOW>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&mdss_dsi_active &mdss_te_active>;
+		port {
+			panel_in: endpoint {
+				remote-endpoint = <&mdss_dsi0_out>;
+			};
+		};
+	};
+};
+
+&mdss_dsi0_out {
+	data-lanes = <0 1 2 3>;
+	remote-endpoint = <&panel_in>;
+};
+
+&mdss_dsi0_phy {
+	vcca-supply = <&vreg_l1b_0p925>;
+	status = "okay";
+};
+
+&mmss_smmu {
+	status = "okay";
+};
+
+&pon_pwrkey {
+	status = "okay";
+};
+
+&pon_resin {
+	status = "okay";
+
+	linux,code = <KEY_VOLUMEDOWN>;
```

Comment:
status should be last property (before possible subnodes) with a blank line before